modules/promotion/src/PromotionConditionManager.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Drupal\commerce_promotion;
+
+use Drupal\Component\Plugin\CategorizingPluginManagerInterface;
+use Drupal\Component\Plugin\Exception\PluginException;
+use Drupal\Core\Plugin\Context\ContextDefinition;
+use Drupal\Core\Cache\CacheBackendInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Executable\ExecutableManagerInterface;
+use Drupal\Core\Executable\ExecutableInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Plugin\CategorizingPluginManagerTrait;
+use Drupal\Core\Plugin\Context\ContextAwarePluginManagerTrait;
+use Drupal\Core\Plugin\DefaultPluginManager;
+
+/**
+ * Provides the Condition plugin manager.
```

Comment:
Bad docblock.

/**
 * Manages discovery and instantiation of promotion condition plugins.
 *
 * @see \Drupal\commerce_promotion\Annotation\CommercePromotionCondition
 * @see plugin_api
 */